ProjFS.Mac/PrjFSKextTests/VnodeCacheTests.mm

```diff
@@ -0,0 +1,174 @@
+#import <XCTest/XCTest.h>
+
+typedef int16_t VirtualizationRootHandle;
+
+#include "../PrjFSKext/VnodeCacheTestable.hpp"
+
+@interface VnodeCacheTests : XCTestCase
+@end
+
+@implementation VnodeCacheTests
+
+struct VnodeCacheEntry
```

Comment:
Should we perhaps move the original definition of `VnodeCacheEntry` into a `VnodeCachePrivate.hpp` file which only `VnodeCache.cpp` *and* the unit tests may include? Maintaining 2 versions seems like a bad idea.

Reply:
I see this conversation is happening in your PR #500 as well (see https://github.com/Microsoft/VFSForGit/pull/500#discussion_r256133349).

I suspect your PR will get in first, and so I'll go with whatever approach we agree on there 😄

Reply:
Closing this comment out, I will go with the `VnodeCachePrivate.hpp` approach